source/frontend/src/features/mapSideBar/acquisition/tabs/expropriation/form9/ExpropriationForm9.tsx

```diff
@@ -0,0 +1,107 @@
+import FormItem from '@/components/common/form/FormItem';
+import { SectionField } from '@/components/common/Section/SectionField';
+import { RestrictContactType } from '@/components/contact/ContactManagerView/ContactFilterComponent/ContactFilterComponent';
+import FilePropertiesTable from '@/components/filePropertiesTable/FilePropertiesTable';
+import { Api_AcquisitionFile } from '@/models/api/AcquisitionFile';
+import { Api_PropertyFile } from '@/models/api/PropertyFile';
+
+import { ExpropriationForm9Model } from '../models';
+import { ExpropriationForm9YupSchema } from './ExpropriationForm9YupSchema';
+
+export interface IExpropriationForm9Props {
+  acquisitionFile: Api_AcquisitionFile;
+  onGenerate: (values: ExpropriationForm9Model) => Promise<void>;
+}
+
+export const ExpropriationForm9: React.FC<IExpropriationForm9Props> = ({
+  acquisitionFile,
+  onGenerate,
+}) => {
+  const onSubmit = async (
+    values: ExpropriationForm9Model,
+    formikHelpers: FormikHelpers<ExpropriationForm9Model>,
+  ) => {
+    try {
+      await onGenerate(values);
+    } finally {
+      formikHelpers?.setSubmitting(false);
+    }
+  };
+
+  const onCancelClick = (formikProps: FormikProps<ExpropriationForm9Model>) => {
+    formikProps.resetForm();
+  };
+
+  const onGenerateClick = (formikProps: FormikProps<ExpropriationForm9Model>) => {
+    formikProps.setSubmitting(true);
+    formikProps.submitForm();
+  };
+
+  return (
+    <Formik<ExpropriationForm9Model>
+      enableReinitialize
+      initialValues={new ExpropriationForm9Model()}
+      validationSchema={ExpropriationForm9YupSchema}
+      onSubmit={onSubmit}
+    >
+      {formikProps => (
+        <React.Fragment>
+          <SectionField label="Expropriation authority" required>
+            <ContactInputContainer
+              field="expropriationAuthority.contact"
+              View={ContactInputView}
+              restrictContactType={RestrictContactType.ONLY_ORGANIZATIONS}
+              displayErrorAsTooltip={false}
+            ></ContactInputContainer>
+          </SectionField>
+          <SectionField
+            label="Impacted properties"
+            required
+            tooltip="For the selected properties - corresponding property and interest details will be captured on the form."
+          >
+            <FormItem field="impactedProperties">
+              <FilePropertiesTable
+                disabledSelection={false}
+                fileProperties={acquisitionFile.fileProperties ?? []}
+                selectedFileProperties={formikProps.values.impactedProperties}
+                setSelectedFileProperties={(fileProperties: Api_PropertyFile[]) => {
+                  formikProps.setFieldValue('impactedProperties', fileProperties);
+                }}
+              ></FilePropertiesTable>
+            </FormItem>
+          </SectionField>
+          <SectionField label="Shown on plan(s)">
```

Comment:
confluence has "Shown on Plan:" not sure which is correct.

Reply:
I was going with the mockup. The sample value indicate that they can enter multiple plan(s) - e.g EPP1234, EPP345656